bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/InstrumentRPCInterceptor.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.config;
```

Comment:
This doesn't belong in the config package, I think com.google.cloud.bigtable.grpc or com.google.cloud.bigtable.metrics would be more appropriate